content/en/docs/concepts/architecture/nodes.md

```diff
@@ -299,3 +306,7 @@ API object can be found at:
 [Node API object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#node-v1-core).
 
 {{% /capture %}}
+{{% capture whatsnext %}}
+* Read about [node components](https://kubernetes.io/docs/concepts/overview/components/#node-components)
```

Comment:
What is the `node components` link you add here? Is this maybe a merge error from a previous version of the file?

Reply:
It was added at the recommendation of @sftim if I recall correctly